lib/pierre.py

```diff
@@ -167,6 +176,12 @@ def get_owner_and_repo(event):
     return owner, repo
 
 
+def get_dependency_id(event):
+    dependency_id = event.get("issue", {}).get("number", "")
```

Comment:
you can change to 
`return event.get("issue", {}).get("number", "")`
no need to assign and then return